<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_mfc">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>771</width>
      <height>481</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>mfc1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>mfc2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>mfc3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>mfc4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>time(sec)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>state</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="select_com">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>76</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_butt">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>420</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_butt">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>490</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_butt">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="connect_led">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="scan_butt">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>SCAN PORT</string>
    </property>
   </widget>
   <widget class="QLabel" name="connect_text">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>hi!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnect_butt">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>50</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_file_butt">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>170</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_file_butt">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>290</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QLabel" name="open_file_label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>140</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>file?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_row_butt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>행 추가</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_row_butt">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>580</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>선택 행 삭제</string>
    </property>
   </widget>
   <widget class="QLabel" name="cell_cursor_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>491</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_cell_butt">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>580</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>테이블 초기화</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="save_file_edittext">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>260</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="done_time_label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>550</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
